.github/workflows/release-latest.yml

```diff
@@ -33,6 +33,15 @@ jobs:
       dockerfile_path: images/openscap/Dockerfile
       vendor: 'Compliance Operator Authors'
 
+  osde2e-push-latest:
+    use: metal-toolbox/container-push/.github/workflows/container-push.yml@main
+    with:
+      name: compliance-operator-osde2e
+      registry_org: complianceascode
+      tag: latest
+      dockerfile_path: Dockerfile.rosa-ci
```

Comment:
Did you forget to add this file?

Reply:
That file was proposed in a separate patch. https://github.com/ComplianceAsCode/compliance-operator/pull/318/files